android/assets/jsons/translations/template.properties

```diff
@@ -644,6 +652,7 @@ Pick [technology] as free tech =
 Units enabled = 
 Buildings enabled = 
 Wonder = 
+Wonder of the World = 
```

Comment:
I'm serious about this. this should be removed, the game already has a clear "Wonder / National Wonder" phrasing, we shouldn't be adding new indicators to things that already exist.

Reply:
OK - was just an attempt to make the entries look cooler:
![image](https://user-images.githubusercontent.com/63000004/126083306-f90857b5-db27-4e11-917c-a1390a2dde37.png)

Not that anybody would have noticed...